opm/simulators/wells/VFPProperties.hpp

```diff
@@ -70,9 +74,33 @@ class VFPProperties {
         return &m_prod;
     }
 
+    double getExplicitWFR(const int table_id, const size_t well_index) const {
```

Comment:
I guess your comment here is related to the first comment about adding the wellState to vfpProperties. My reasoning behind adding it and not passing it around is to avoid a minor feature like this to pollute all the call places. To pass the explicit value around would require changes to significant number of interfaces.